docs/config-janitor.rst

```diff
@@ -47,12 +47,17 @@ Any configuration key/value listed here may also be used in the specific plugin
 
     `Optional`: Timeout in seconds for how long each plugin should wait for outstanding tasks (e.g. processing remaining message from a channel) before cancelling. This is only used when a plugin has received all messages from a channel, but may have work outstanding. Defaults to ``60``.
 
+.. option:: default_zone_prefix="STR"
+
+    `Optional`: Prefix associated with Google managed zone names, prepended with a '-' to the generated name.
```

Comment:
I think the phrasing is a little unclear.  'prepended to the generated name with a - in between' and then an example, maybe?

Reply:
I think the between makes it more confusing, but I have added an example.